### PR TITLE
Ensemble Simulate minor tweaks

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description
- a missing await was causing some ugly errors in the node page
- remove timespan from state and only utilize endTime. start will always be 0.
- update `solver method` and `step size`

# Required:
https://github.com/DARPA-ASKEM/pyciemss-service/pull/126

### CODE REVIEW

**Review by blanchco — APPROVED** [review]

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss-drilldown.vue [hunk 1]
```diff
@@ -165,6 +165,10 @@
 											:options="[CiemssMethodOptions.dopri5, CiemssMethodOptions.euler]"
 										/>
 									</div>
+									<div v-if="knobs.method === CiemssMethodOptions.euler" class="label-and-input">
+										<label for="num-steps">Solver step size</label>
+										<tera-input-number inputId="integeronly" v-model="knobs.stepSize" />
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss-drilldown.vue:170]: - This `<label for=".." ...`  will throw a warning if there is no target, I would probably just remove the `for` attribute
- I don't know what `inputId` does, but there is nothing saying step-size cannot be fractional

Comment (mwdchang) [reply packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss-drilldown.vue:170]: eg this thing
image

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss-drilldown.vue:170]: Thank you

**Pull request merged**